docker/scripts/db-init/db_init.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+echo "Creating DB '$POSTGRES_DB' and schema '$POSTGRES_SCHEMA' with user '$POSTGRES_USER'"
+
+psql -U ${POSTGRES_USER} -d ${POSTGRES_DB}<<-END
+    ALTER ROLE ${POSTGRES_USER} SET client_encoding TO 'utf8';
+    ALTER ROLE ${POSTGRES_USER} SET default_transaction_isolation TO 'read committed';
```

Comment:
@chandrasekharan-zipstack why moved this into a shell script?.  Any *.sql files will be executed by POSTGRES_USER, which defaults to the postgres superuser.